python/sklearnserver/sklearnserver/__main__.py

```diff
@@ -31,6 +32,7 @@
 args, _ = parser.parse_known_args()
 
 if __name__ == "__main__":
+    Storage.wait_for_model_data(args.model_dir)
```

Comment:
I think if this is not the right place for waiting the model as it only applies to KServe implemented model runtimes, we need to make sure it works for all supported serving runtimes including triton, torch serve etc. So I think we need some other ways to ensure when model server starts the model data is already mounted and can be loaded into the main memory.

Reply:
Thanks, let revert the Python changes, as they are out of scope of this PR. One can consider this fixing dependency (e.g. that a model might not be immediately available during startup) in a latter PR.

Reply:
Removed in latest commit.